docs/snippets/ie_common.py

```diff
@@ -5,8 +5,8 @@
 
 #! [ie:create_core]
 import numpy as np
-import openvino.inference_engine as ie
-core = ie.IECore()
+import openvino as ov
+core = ov.inference_engine.IECore()
```

Comment:
let's keep old API code as is (it will be removed in 2024.0)

Reply:
Reverted it with the https://github.com/openvinotoolkit/openvino/pull/19084/commits/b4ead7997cfa46f091d33c5ca3cb3d3dbcd207a5 commit